### ISSUE TITLE
Minimal SMT creation functionality

### ISSUE THREAD

**Issue by theoreticalbts** [steemit/steem#1681]
We need a simple way to create an SMT asset with an initial SMT balance sent to the creator.  If the existing SMT creation ops can do this already, great, let's put the steps in an example script and close this ticket.

If we don't have this functionality, let's define a temporary new operation or extension to implement it quickly for testing purposes.  It can be removed later.

**Comment by Kiwonik** [comment]
Here's the minimal list of smt operations that need be implemented to achieve the stage of "initial SMT balance sent to the creator":
1. `smt_create_operation`
1. `smt_setup_operation`
1. `smt_contribute_operation`

We skip over inflation/parameters setup. All caps are assumed public, hence no need for reveal operation. Also sufficient contributions are assumed to avoid refund op implementation.

**Comment by Kiwonik** [comment]
The three critical operations status is as follows:
1. `smt_create_operation` - would be complete were it not for necessary and promised methods  `claim_smt_next_identifier` and `get_smt_next_identifier`. See #1554
1. `smt_setup_operation` - work in progress - we're about to implement an efficient timed event scheduler. See #1724
1. `smt_contribute_operation` - suspended with 'question' status. See #1694

**Comment by youkaicountry** [comment]
(1), I have created a new issue for `claim_smt_identifier` #1777, and assigned it to @theoreticalbts. I have also assigned him to #1633. 
For now we will move directly ahead with `claim_smt_identifier` to unblock progress.
(2), For now all we want to move ahead with bottom-up development is to create an SMT and give a balance to a user, but not necessarily in the way it will be done in production. The `smt_setup_operation` step does not need to be implemented at this point.
(3), As with (2), this is not needed to create an SMT, and give a balance to a user. This is not a blocking issue at this point.
